Realm/RLMMongoCollection.h

```diff
@@ -18,12 +18,27 @@
 
 #import <Foundation/Foundation.h>
 #import <Realm/RLMObjectId.h>
+#import <Realm/RLMNetworkTransport.h>
 
 NS_ASSUME_NONNULL_BEGIN
 @protocol RLMBSON;
 
 @class RLMApp, RLMFindOptions, RLMFindOneAndModifyOptions, RLMUpdateResult;
 
+/// Delegate which is used for subscribing to changes a  `watch` stream.
+@protocol RLMChangeEventDelegate
+/// The stream was opened.
+- (void)didOpen;
```

Comment:
These should be named more explicitly: `changeStreamDidOpen:(RLMChangeStream)` `changeStream:(RLMChangeStream) didCloseWithError:(nullable NSError *)` etc. Whatever `RLMChangeStream` is should have some basic closing functionality.